tests/MyGame/Example/Monster.go

```diff
@@ -503,20 +503,40 @@ type Monster struct {
 	_tab flatbuffers.Table
 }
 
+const MonsterIdentifier = "MONS"
+
 func GetRootAsMonster(buf []byte, offset flatbuffers.UOffsetT) *Monster {
 	n := flatbuffers.GetUOffsetT(buf[offset:])
 	x := &Monster{}
 	x.Init(buf, n+offset)
 	return x
 }
 
+func FinishMonsterBuffer(builder *flatbuffers.Builder, offset flatbuffers.UOffsetT) {
```

Comment:
`FinishMonsterBufferWithFileIdentifier`  might be more clear here

Reply:
I wanted to follow the C++ API here. The documentation https://flatbuffers.dev/flatbuffers_guide_writing_schema.html says

> For any schema that has such an identifier, flatc will automatically add the identifier to any binaries it generates (with -b), and generated calls like FinishMonsterBuffer also add the identifier. If you have specified an identifier and wish to generate a buffer without one, you can always still do so by calling FlatBufferBuilder::Finish explicitly.

So it's pretty clear to me that the function should be called `FinishMonsterBuffer` and that it should automatically add the `file_identifier`. This is also the way it's implemented in C++ and C#.

Reply:
Ah I see LGTM then.